public/js/client.js

```diff
@@ -9802,3 +9954,167 @@ function sanitizeXSS(src) {
 function disable(elem, disabled) {
     elem.disabled = disabled;
 }
+
+//  Carousel control button click handlers  ---[START]------->
+async function handleDisplayModePrevItem() {
+    if (currentItemIdx === 0) {
+        const propList = await idbKeyval.get(PROPERTY_LIST);
```

Comment:
It's hard to understand where idbKeyval comes from, I had to invest a few minutes of research to find that it comes from a library.